### DISCUSSION TITLE
deploying a Discord bot

### DISCUSSION THREAD

**Discussion by ankush802135** [vercel/community#3116] [Help]
### Summary

hey, I'm new at vercel,and i made a discord bot using discord.py and just want to know the steps that how to deploy my bot frok GitHub repo

### Steps to Reproduce

i don't know, just want to know steps

**Comment by maddsua** [comment]
> ### Summary
> 
> hey, I'm new at vercel,and i made a discord bot using discord.py and just want to know the steps that how to deploy my bot frok GitHub repo
> ### Example
> 
> _No response_
> ### Steps to Reproduce
> 
> i don't know, just want to know steps

Does it have a frontend UI? If not, you've picked the wrong platform. I'd recommend looking towards Railway or DigitalOcean. Vercel isn't suited for service-like apps (ones that need to run continuously, not on-demand functions)

**Comment by Chin-ma** [comment]
You can use Heroku or AWS but you will have to pay for it
